docs/api/1.0.0-SNAPSHOT.md

```diff
@@ -0,0 +1,145 @@
+# API Docs - v1.0.0-SNAPSHOT
```

Comment:
Remove this file

Reply:
Removed the file. Commit f44a2ca50606f5046df52930a8396ec13cbde73e